### ISSUE TITLE
Feature Request: Allow to filter linting files

### ISSUE THREAD

**Issue by sergei-startsev** [dequelabs/axe-linter-action#49]
## Feature Overview
Axe Linter action should allow to filter files to lint to reduce the noise of false-positive errors.

## Motivation
Axe-linter may misinterpret HTML template files, resulting in false positive issues.

### Example 1
The lodash template below causes a false-positive `aria-valid-attr-value - Ensures all ARIA attributes have valid values` error:

```html
<li role="tab" aria-selected="<%-param1 %>" aria-controls="<%- param2 %>">Tab Name</li>
```

### Example 2
The lodash template below causes a false-positive `list - Ensures that lists are structured correctly` error:

```html
<ul>
    <% for (let i = 0; i < array.length; i++){%>
         <!--some code goes here-->
    <% } %>
</ul>
```

**Comment by ballendq** [comment]
Hi @sergei-startsev the linter does not support template files, this is why you are seeing problems.

To reduce noise in results, we recommend you exclude files using the [exclude option](https://docs.deque.com/linter/4.0.0/en/axe-linter-configuration#exclude) within your configuration file.

**Comment by sergei-startsev** [comment]
Reopened the issue since it's a feature request

**Comment by sergei-startsev** [comment]
Closed in favor of #50
